os/lib/dbg-io/strformat.c

```diff
@@ -373,6 +373,17 @@ format_str_v(const strformat_context_t *ctxt, const char *format, va_list ap)
       } else {
         flags |= SIZE_SHORT;
       }
+    } else if(*pos == 'z') {
+      if(sizeof(size_t) == sizeof(short)) {
+        flags |= SIZE_SHORT;
+      } else if(sizeof(size_t) == sizeof(long)) {
+        flags |= SIZE_LONG;
+#ifdef HAVE_LONGLONG
+      } else if (sizeof(size_t) == sizeof(long long)) {
```

Comment:
Remove space between if and opening paren

Reply:
Well spotted! Perhaps if I setup my environment better some automated tool would have helped me realizing this before pushing. Anyway. Now it is fixed.